membership-common/src/main/scala/com/gu/memsub/BillingSchedule.scala

```diff
@@ -36,13 +36,13 @@ case class BillingSchedule(invoices: NonEmptyList[Bill]) {
 
 object BillingSchedule {
 
-  case class BillItem(name: String, product: Option[Benefit], amount: Float, unitPrice: Float) {
-    override lazy val toString = s"| $amount\t| ($unitPrice)\t| $name | ${product.mkString}"
+  case class BillItem(name: String, amount: Float, unitPrice: Float) {
+    override lazy val toString = s"| $amount\t| ($unitPrice)\t| $name"
```

Comment:
this is the toString that used to use the product from the catalog.  But although this might be useful for debugging, it didn't seem worth passing it all the. way through.